core/src/main/java/org/infinispan/configuration/parsing/Parser80.java

```diff
@@ -1083,6 +1107,31 @@ protected void parseCacheElement(XMLExtendedStreamReader reader, Element element
             this.parseExpiration(reader, builder);
             break;
          }
+         case DATA_CONTAINER: {
+            parseDataContainer(reader, holder);
+            break;
+         }
+         case SECURITY: {
+            this.parseCacheSecurity(reader, builder);
+            break;
+         }
+         default: {
+            reader.handleAny(holder);
+         }
+      }
+   }
+
+   protected void parseFullCacheElement(XMLExtendedStreamReader reader, Element element, ConfigurationBuilderHolder holder) throws XMLStreamException {
```

Comment:
simple cache is-a cache, full cache is-a cache (and local, dist and repl are full caches), but full-cache is-not-a simple cache, therefore it wouldn't be correct...
But ok, I can probably use `parseBaseCacheElement`...